src/common/options.js

```diff
@@ -223,6 +275,18 @@ const options = {
     actionBtnClass:"UNX-action-item",
     actionChangeClass:"UNX-action-change",
     onAction: function(e,ctx) {
+    },
+    onQueryRedirect:(self, redirect)=>{
+        if(redirect) {
+            const {
+                value,
+                type
+            } = redirect;
+            if(type === "url") {
+                location.href =  value;                                                           
+            }
+            return false;
+        }
     }
    // searchQueryParam:null
 };
```

Comment:
Can we please add all these new configs in the github Readme file

Reply:
done